composer.json

```diff
@@ -2,15 +2,15 @@
   "name": "magento/composer",
   "description": "Magento composer library helps to instantiate Composer application and run composer commands.",
   "type": "library",
-  "version": "1.0.3",
+  "version": "1.2.0",
```

Comment:
There is already v 1.2.0 released. See https://github.com/magento/composer/releases
As I understand, this updated version should be used for Magento 2.2 release line (which uses `magento/composer `1.2.0` currently). I propose to base the branch on `1.2.0` release tag (it already has `composer/composer` `1.4.1` for example, and release new version as `1.2.1` (`1.3.0` is already in use and has higher versions of libraries).

Reply:
done